### PR TITLE
[FE] member management UI

### PR DESCRIPTION
**Pull request by tinophan247**
dev testing result:
2022-12-15_15h41_00
2022-12-15_15h40_51

### CODE REVIEW

File: src/views/components/TextField/index.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                    />
+                    :
+                    <TextField
+                        size='small'
+                        value={value}
+                        onChange={onChange}
+                        type={type}
+                    />
+                }
+
```
Comment (phantuananh18) [inline src/views/components/TextField/index.jsx:35]: Remove redundant line 35

File: src/views/pages/AdminSide/MemberManagement/MemberModal.jsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+    useEffect(() => {
+        setOpen(isShow);
+    }, [isShow]);
+
+    useEffect(() => {
+        setFormState({ ...data });
+    }, [data]);
+
+
```
Comment (phantuananh18) [inline src/views/pages/AdminSide/MemberManagement/MemberModal.jsx:46]: remove redundant line 46

File: src/views/pages/AdminSide/MemberManagement/MemberTable.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+import { StyledTableCell } from './style';
+import MemberModal from './MemberModal';
+import MemberTR from './MemberTR';
+import { defaultMember } from '../../../../constants';
+
+const MemberTable = () => {
+    const [showModal, setShowModal] = useState(false)
+    const [memberProfile, setMemberProfile] = useState(defaultMember)
+
+    const createData = (fullname, username, role, groupName, email, phoneNumber, DOB, weight, gender) => {
```
Comment (phantuananh18) [inline src/views/pages/AdminSide/MemberManagement/MemberTable.jsx:17]: Format bracket here

File: src/views/pages/AdminSide/MemberManagement/MemberTR.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { StyledTableCell, StyledTableRow } from './style';
+
+const MemberTR = ({ item, handleOpen }) => {
```
Comment (phantuananh18) [inline src/views/pages/AdminSide/MemberManagement/MemberTR.jsx:4]: What is mean "MemberTR" ? You should rename it clearly.

**Review by phantuananh18 — COMMENTED** [review]
Check my comment

File: src/views/pages/AdminSide/MemberManagement/MemberTR.jsx (hunk 4 above)
Comment (phantuananh18) [reply src/views/pages/AdminSide/MemberManagement/MemberTR.jsx:4]: Thanks for your comment.

**Comment by tinophan247** [comment]
> Check my comment

Resolved all

**Review by phantuananh18 — APPROVED** [review]

**Review by ducvp0702 — APPROVED** [review]

**Pull request merged**